src/classes/TDTM_DefaultConfig.cls

```diff
@@ -352,6 +352,10 @@ public without sharing class TDTM_DefaultConfig {
                 Class__c = 'CRLP_Rollup_TDTM', Load_Order__c = 2, Object__c = 'npe01__OppPayment__c',
                 Trigger_Action__c = 'AfterInsert;AfterUpdate;AfterDelete;AfterUndelete'));
 
+        handlers.add(new Trigger_Handler__c(Active__c = true, Asynchronous__c = false,
```

Comment:
@force2b @RadSFDeveloper 

Since we're adding a new trigger for GAU and `BeforeDelete` for RDs, what's the best way to make sure that customers that are already have TDTM installed get these records updated correctly? Should we query for these records in the Post Install class and update if they exist?

Reply:
There is already a process for this in the `STG_InstallScript.insertTdtmDefaults()` method.